calico-enterprise/operations/monitor/metrics/recommended-metrics.mdx

```diff
@@ -0,0 +1,514 @@
+
+### Iptables save errors
+
+| Iptables save errors             |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *sum(rate(felix_iptables_save_errors[5m]))*                  |
+| Explanation                      | Number of iptables-save errors. The iptables-save command is run before every iptables-restore command so that {{prodname}} has the current state of iptables. |
+| Threshold value recommendation   | Occasional errors are normal.  Investigate error counters that rapidly rise. |
+| Threshold breach symptoms        | {{prodname}} network policies are not up to date. Cluster nodes may be under resource contention, which may result in other _error and _seconds metrics rising.  Repeated errors could mean some persistent problem (e.g. some other process is creating iptables rules that {{prodname}} cannot decode with the version of iptables-save in use). |
+| Threshold breach recommendations | See the Errors Plot graph to determine if the scope is cluster-wide or node-local.  Check calico-node logs. Check resource usage and contention on Kubernetes nodes and calico-nodes.  Add nodes/resources if needed.  If no resource contention is seen, restart calico-node and monitor. |
+| Priority level                   | Optional.                                                    |
+
+### Felix log errors
+
+| Felix log errors                 |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *sum(rate(felix_log_errors[5m]))*                            |
+| Explanation                      | The number of times Felix fails to write out a log because the log buffer is full. |
+| Threshold value recommendation   | Occasional errors are normal.  Investigate error counters that rapidly rise. |
+| Threshold breach symptoms        | Calico-node may be under resource contention, which may result in other _errors and _seconds metrics rising. |
+| Threshold breach recommendations | See the Errors Plot graph to determine if the scope is cluster-wide or node-local.  Check resource usage and contention on Kubernetes nodes and calico-nodes.  Add nodes/resources if needed.  If no resource contention is seen, restart calico-node and monitor. |
+| Priority level                   | Optional.                                                    |
+
+### Errors plot
+
+| Errors plot                      |                                                              |
+| -------------------------------- | ------------------------------------------------------------ |
+| Metric                           | *rate(felix_ipset_errors[5m]) \|\| rate(felix_iptables_restore_errors[5m]) \|\| rate(felix_iptables_save_errors[5m]) \|\| rate(felix_log_errors[5m])* |
```

Comment:
Not really a metric, more of a recommendation for how to monitor these metrics using a graph

Reply:
Changed to: Monitor metrics using a graph